news/772.misc

```diff
@@ -0,0 +1 @@
+Fix a packaging issue (missing sdist dependency)
```

Comment:
It makes sense to have this same fragment on both `2.1_branch` and `master`, right?

Reply:
No, I think we should only report it once.
Remove it from the cherry pick.

Reply:
Done in https://github.com/omry/omegaconf/pull/777/commits/c68fd10cec667fd0222b6808b80c6bc7e475aaf1.